README.md

```diff
@@ -6,16 +6,12 @@
 
 ## READ FIRST!!
 
-- **Check the [CHANGELOG.rst](https://github.com/StackStorm/st2-docker/blob/master/CHANGELOG.rst)** file for any potential
-  changes that may require restarting containers.
+- **Check the [CHANGELOG.rst](https://github.com/StackStorm/st2-docker/blob/master/CHANGELOG.rst)**
+  file for any potential changes that may require restarting containers.
 - Be sure to use the latest `docker-compose.yml`. Run `git pull` in your `st2-docker` workspace!
 - Run `st2ctl reload --register-all` to reload all services.
-- **For information on how the stackstorm docker image is versioned, see [VERSIONING.md](https://github.com/StackStorm/st2-docker/blob/master/VERSIONING.md)**
-- If a specific image is required, it is always best to be explicit and specify the Image ID. For example:
```

Comment:
@vutny I guess it is still reasonable to put this note on top of the README.md so maybe we can say:

If a specific image is required, it is always best to be explicit and specify the image digest. See `ST2_IMAGE_TAG` for example.